webui/react/src/pages/TrialDetails/Header/TrialHeaderLeft.tsx

```diff
@@ -14,6 +17,9 @@ interface Props {
   trial: TrialDetails;
 }
 
+const labelMaxLength = 12;
```

Comment:
If the tooltip max length is 12, do you think the same restriction should go to exp config so the user gets an error message when the enter a signal that is longer than 12 characters?

Reply:
I don't really have a strong opinion, here we just truncate whatever is longer than 12, but we can certainly have restriction on DB level

Reply:
Cool, I will do in either exp config or db level.

Reply:
Oh I might have misunderstood the code. So the limitation applies to the badge but the tooltip shows the full text?

Reply:
Exactly